src/Web/Sfa.Das.Sas.Web.UnitTests/Application/Handlers/SitemapHandlerTest.cs

```diff
@@ -109,5 +111,59 @@ public void ShouldReturnXmlSitemapWithCorrectLinksToIndividualNonEmployerProvide
             nodes.ElementAt(0).Value.Should().Be($"http://localhost/providers/{ukprnToProcess}/{providerNameProcessed}");
         }
 
+        [Test]
+        public void ShouldReturnXmlSitemapWithStandardNames()
+        {
+            var standardOneTitle = "Standard One";
+            var standardOneEncoded = "standard-one";
+
+            _mockGetStandards.Setup(x => x.GetAllStandards()).Returns(new List<Standard>
+            {
+                new Standard {StandardId = "1", Title = standardOneTitle, IsPublished = true},
+                new Standard {StandardId = "2", IsPublished = true}
```

Comment:
maybe add another standard here where IsPublished = false?   I don't think the test will need changing, as that should not appear in the response, right?

Reply:
I have added a separate test with an unpublished Standard to show that it isn't retrieved.